package.json

```diff
@@ -54,10 +55,12 @@
     "@vue/test-utils": "^1.0.0-beta.29",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.2",
+    "chromedriver": "78",
```

Comment:
@moonming  We have some new dependencies for E2E test cases, so if I need to provide the License for them?

Reply:
I think you can discuss license issues in mailing list.
 IMO, we only need add license for bundle codes in repo, not dependencies.

Reply:
it's not need to add license if dependencies not in release packaege.

Reply:
I see